### ISSUE TITLE
Can you tell me the hyperparameters you use in the FedSEM algorithm?

### ISSUE THREAD

**Issue by moFang222** [anonymousgit2020/fedsem#2]
Hi! Great work of coming up with this fedsem algorithm!
I tried to reproduce the results in the paper,but I encountered the problem of low accuracy(The accuracy of fedsem and fedavg algorithm in the original paper is very high).
So I very want to know what hyperparameters did you use during training and how did you divide the data set since they are not specifically mentioned in the paper.Many thanks!
